packages/dashboards/addon/templates/common/navi-loader.hbs

```diff
@@ -1,6 +1,2 @@
 {{!-- Copyright 2017, Yahoo Holdings Inc. Licensed under the terms of the MIT license. See accompanying LICENSE.md file for terms. --}}
-<div class="spinner">
-  <div class="bounce1"></div>
-  <div class="bounce2"></div>
-  <div class="bounce3"></div>
-</div>
+<DenaliLoader />
```

Comment:
Hmm, can we remove this one. I am not sure this is used.

Reply:
Doesnt seem to be used, I removed it and ran the dashboards tests with no fails